CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+<!--- Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com> -->
+# Lagom contributor guidelines
+
+## Prerequisites
+
+Before making a contribution, it is important to make sure that the change you wish to make and the approach you wish to take will likely be accepted, otherwise you may end up doing a lot of work for nothing.  If the change is only small, for example, if it's a documentation change or a simple bug fix, then it's likely to be accepted with no prior discussion.  
```

Comment:
Can we point people to Gitter like we do in the [`lagom/lagom` contributing guide](https://github.com/lagom/lagom/blob/master/CONTRIBUTING.md#prerequisites)? (Maybe Discourse would be better, for both of them.)

Reply:
+1